pkg/controllers/webhook/controller.go

```diff
@@ -487,9 +487,14 @@ func (c *controller) updatePolicyStatuses(ctx context.Context) error {
 	}
 	for _, policy := range policies {
 		if policy.GetNamespace() == "" {
-			_, err := controllerutils.UpdateStatus(
+			p, err := c.kyvernoClient.KyvernoV1().ClusterPolicies().Get(ctx, policy.GetName(), metav1.GetOptions{})
```

Comment:
We are making a get request for every policy, one at a time. Is it possible to create one get request that fetches all the policies and cluster policies?

Reply:
The issue is that other threads update the policy thus it returns "object has been modified". I tried getting the policy using the lister but that didn't solve the issue, there seems to be a delay in the cache to refresh items.